api/schema_responses/serializers.py

```diff
@@ -0,0 +1,108 @@
+from api.base.serializers import (
+    JSONAPISerializer,
+    LinksField,
+    RelationshipField,
+)
+
+from rest_framework import serializers as ser
+from api.base.utils import absolute_reverse
+from osf.models.schema_responses import SchemaResponses
+from osf.models import Registration
+from rest_framework import exceptions
+from django.utils import timezone
+
+
+class SchemaResponsesSerializer(JSONAPISerializer):
+    id = ser.CharField(required=False, source='_id', read_only=True)
+    title = ser.CharField(required=False, allow_blank=True)
+    responses = ser.JSONField(required=False)
+    deleted = ser.SerializerMethodField(required=False)
+    public = ser.SerializerMethodField(required=False)
+
+    def get_deleted(self, obj):
+        return bool(obj.deleted)
+
+    def get_public(self, obj):
+        return bool(obj.public)
+
+    links = LinksField(
+        {
+            'self': 'get_absolute_url',
+        },
+    )
+
+    node = RelationshipField(
```

Comment:
can we just call this `registration` instead of `node`, which means basically anything?